diff --git a/index.d.ts b/index.d.ts
index 36906e5bf4f37c3ab21bde4ce15ffd92756de5b4..35f4653815d8facd630214e0c4c5b8de0e233551 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -7,7 +7,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 4.3
 
-/// <reference types="jest" />
+// See https://github.com/testing-library/jest-dom/issues/427 for reference
 
 import { TestingLibraryMatchers } from './matchers';
 