diff --git a/index.d.ts b/index.d.ts
index 17be53259f5b8702627371c3205586fe4a62a57d..37ac9e304b81b2a97804bd9abbdb27ae8f442562 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -7,7 +7,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 4.3
 
-/// <reference types="jest" />
+// See https://github.com/testing-library/jest-dom/issues/427 for reference
 
 import { TestingLibraryMatchers } from './matchers';
 